spec/support/features.rb

```diff
@@ -7,11 +7,23 @@ def in_spec?
 end
 
 def with_env_vars_runner(env)
+  # hold on to old values
   env = env.stringify_keys
```

Comment:
Of _course_ There's A Thoughtbot Project For That :wink: [Storied](https://www.pivotaltracker.com/story/show/105137620).